lib/alerts.py

```diff
@@ -20,6 +16,10 @@ def current(self):
     def expired(self):
         return [alert for alert in self if alert.is_expired]
 
+    @property
+    def last_updated(self):
+        return max(alert.starts for alert in self if alert.is_current)
```

Comment:
Minor: we could test to show that we expect to get an exception here in the case there are no alerts (since this code shouldn't be used in that scenario).

Reply:
Added test in f04c6ff